.github/workflows/ci.yml

```diff
@@ -152,8 +152,7 @@ jobs:
     - name: Run cargo clippy
       uses: actions-rs/cargo@v1.0.3
       with:
-        command: clippy
-        args: -- -A clippy::type_complexity -A clippy::pedantic -A clippy::style
+        command: lint # cargo alias to allow reuse of config locally
```

Comment:
Mhh, do I understand correctly that the alias would be needed to ease local development, in other words in order not to have to type `clippy -- -A clippy::type_complexity -A clippy::pedantic -A clippy::style -D warnings` before commiting? Though not particularly important, I don't think the additional convenience is worth the indirection, especially since I don't expect contributors to run into many clippy lints.

Reply:
> since I don't expect contributors to run into many clippy lints.

The issue is that without specifying `-A clippy::type_complexity -A clippy::pedantic -A clippy::style` clippy returns something like 200 results for rust-libp2p :)

In case of a CI failure, users need to specify this entire configuration to reproduce the errors locally. Once clippy ships its configuration file, I think we should switch to that but until then, I see this alias as the best option we have to make fixing linting errors locally reasonably smooth.

Reply:
:+1: sounds good to me. Thanks for the additional reasoning.

I would just suggest renaming `lint` to something like `custom-clippy` or something along those lines.